src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -941,6 +941,31 @@ public void canSearchForInstancesUsingSimilarQueryToUILookAheadSearch() {
     canSort("title=\"upr*\" or contributors=\"name\": \"upr*\" or identifiers=\"value\": \"upr*\"", "Uprooted");
   }
 
+  @Test
+  public void arrayModifierfsContributors1() {
+    canSort("contributors = /@name novik", "Uprooted", "Temeraire" );
+  }
+
+  @Test
+  public void arrayModifierfsContributors2() {
+    canSort("contributors = /@contributorNameTypeId = personal novik",  "Uprooted", "Temeraire");
+  }
+
+  @Test
+  public void arrayModifierfsIdentifiers1() {
+    canSort("identifiers = /@value 9781447294146", "Uprooted");
+  }
+
+  @Test
+  public void arrayModifierfsIdentifiers2() {
+    canSort("identifiers = /@identifierTypeId = isbn 9781447294146", "Uprooted");
+  }
+
+  @Test
+  public void arrayModifierfsIdentifiers3() {
+    canSort("identifiers = @identifierTypeId asin", "Nod");
```

Comment:
No slash before @ like other test cases?

Reply:
Also there is no "=" before "asin". Does the new syntax support that?

Reply:
The omitted slash was a mistake.. But it shows that full text search in many cases.. (key+values in sequence gets a hit)

Reply:
It's simply a search identifiers = "@identifierTypeId asin" .. @ most likely will be ŕemoved.. ANd "identitifierTypeId asin" is left with two tokens in order .. (phrase).